docs/examples/contrib/sqlalchemy/sqlalchemy_async_repository.py

```diff
@@ -19,42 +19,43 @@
     AsyncSessionConfig,
     SQLAlchemyAsyncConfig,
     SQLAlchemyInitPlugin,
-    base,
-    filters,
-    repository,
 )
+from litestar.plugins.sqlalchemy.base import UUIDAuditBase, UUIDBase
+from litestar.plugins.sqlalchemy.filters import LimitOffset
+from litestar.plugins.sqlalchemy.repository import SQLAlchemyAsyncRepository
 
 if TYPE_CHECKING:
     from sqlalchemy.ext.asyncio import AsyncSession
 
 
 class BaseModel(_BaseModel):
-    """Extend Pydantic's BaseModel to enable ORM mode"""
+    """Extended Pydantic's `BaseModel` to enable the ORM mode."""
 
     model_config = {"from_attributes": True}
 
 
-# The SQLAlchemy base includes a declarative model for you to use in your models.
-# The `UUIDBase` class includes a `UUID` based primary key (`id`)
-class AuthorModel(base.UUIDBase):
-    # we can optionally provide the table name instead of auto-generating it
+# The SQLAlchemy base includes a declarative model for you to use in your
+# models. The `UUIDBase` class includes an `UUID` based primary key (`id`).
```

Comment:
*a* is actually correct here. "n before a vowel" refers to the vowel *sound*, not whether the word starts with a letter that's a vowel. This applies to most of the changes made in this PR.

https://www.scribbr.com/commonly-confused-words/a-vs-an/